tests/integration/security/authorization_test.go

```diff
@@ -87,34 +87,39 @@ func TestAuthorization_mTLS(t *testing.T) {
 				ctx.NewSubTest(fmt.Sprintf("From %s", cluster.StableName())).Run(func(ctx framework.TestContext) {
 					a := apps.A.Match(echo.InCluster(cluster).And(echo.Namespace(apps.Namespace1.Name())))
 					c := apps.C.Match(echo.InCluster(cluster).And(echo.Namespace(apps.Namespace2.Name())))
```

Comment:
Looping over `ctx.Clusters()` isn't the best approach. 

1. You can iterate over `app.A`, `app.B`, etc. 
2. The `echotest` framework should be doing enumeration of test cases if possible. 

I understand this is an incremental change to gain VM coverage, so this is fine for now.